extensions/qbittorrent/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "qbitorrent",
+  "title": "QBitorrent",
+  "description": "a qBitorrent client",
+  "icon": "qbittorrent.png",
+  "author": "J3n5en",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "torrents",
+      "title": "List Torrents",
+      "subtitle": "qBittorrent",
+      "description": "list qBitorrent torrents",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "address",
+      "title": "address",
+      "description": "qBitorrent webui address",
+      "placeholder": "http://example.com:8080",
+      "type": "textfield",
+      "required": true
+    },
+    {
+      "name": "username",
+      "title": "username",
+      "description": "qBitorrent webui username",
+      "placeholder": "admin",
+      "type": "textfield",
+      "required": true
+    },
+    {
+      "name": "password",
+      "title": "password",
+      "description": "qBitorrent webui password",
+      "type": "password",
+      "placeholder": "adminadmin",
+      "required": true
+    },
+    {
+      "name": "timeout",
```

Comment:
Would maybe be better to add value in seconds, value under 1000ms will spam everything 😄